### ISSUE TITLE
Fixing Mobile S breakpoint for news layout

### ISSUE THREAD

**Issue by ryangchung** [CarletonComputerScienceSociety/website#506]
As a follow-up of #504 , the tiles of the news items have a breakpoint set too big, and we should accommodate for smaller screens (smallest should be 320px according to accessibility best practices)

![image](https://github.com/CarletonComputerScienceSociety/website/assets/87027981/eefb59ce-c3d4-4363-bf34-326c1f0d51ac)

Revise the tile layout so that it fits and scales well down to 320px.
